Linkedlst.rb

```diff
@@ -0,0 +1,92 @@
+class Node
+
+    attr_accessor :value, :next_node
+
+    def initialize val,next_node
+        @value = val
+        @next_node = next_node
+        puts  value.to_s 
+    end
+end
+
+class LinkedList
+
+    def initialize val
+        @head = Node.new(val,nil)
```

Comment:
In this case, the constructor should not take any arguments as inputs.  When a new linked list is defined, the head points to nil/null